static_test_driver/static_test_driver.ino

```diff
@@ -207,7 +214,6 @@ void loop() {
       for (int i = 0; i < NUM_HIST_VALS; i++)
```

Comment:
Does this mean I can delete all code that has to do with NUM_HIST_VALUES, i.e READ(flag_zero) ?